Language/Structure/Control Structure/while.adoc

```diff
@@ -69,7 +70,7 @@ while (var < 200) {
 === See also
 
 [role="language"]
-
+ https://www.arduino.cc/reference/en/language/structure/control-structure/break/[Break^]
```

Comment:
```suggestion
* #LANGUAGE# `link:../break[break]`

```
- Make clickable relative link
- Follow [standard style conventions](https://raw.githubusercontent.com/arduino/reference-en/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc) for reference links